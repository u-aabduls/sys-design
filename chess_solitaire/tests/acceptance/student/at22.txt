-- ACCEPTANCE TEST #22: VIOLATION CASE: moves - game_started

-- Setup some chess pieces for the game
setup_chess (K,1,3)
setup_chess (Q,2,4)
setup_chess (B,2,3)
setup_chess (R,4,1)

-- attempt to move_and_capture while game hasn't started
move_and_capture(-10,10,0,5)