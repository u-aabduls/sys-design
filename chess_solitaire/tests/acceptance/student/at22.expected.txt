  # of chess pieces on board: 0
  Game being Setup...
  ....
  ....
  ....
  ....
->setup_chess(K,1,3)
  # of chess pieces on board: 1
  Game being Setup...
  ..K.
  ....
  ....
  ....
->setup_chess(Q,2,4)
  # of chess pieces on board: 2
  Game being Setup...
  ..K.
  ...Q
  ....
  ....
->setup_chess(B,2,3)
  # of chess pieces on board: 3
  Game being Setup...
  ..K.
  ..BQ
  ....
  ....
->setup_chess(R,4,1)
  # of chess pieces on board: 4
  Game being Setup...
  ..K.
  ..BQ
  ....
  R...
->move_and_capture(-10,10,0,5)
  # of chess pieces on board: 4
  Error: Game not yet started
  ..K.
  ..BQ
  ....
  R...
